pulumi/src/cluster/docker.ts

```diff
@@ -43,21 +35,17 @@ export const buildAndPushDockerImages = async (dockerhub: Dockerhub, name: strin
   }
 
   const volumeReaperImage = `${dockerhub.username}/${name}-volume-reaper`
-  const volumeReaperHash = await getVolumeReaperHash()
+  const volumeReaperTag = await getVolumeReaperHash()
```

Comment:
use same naming convention (realize it does conflict with `Hash` function naming, but matches `tags` usage below)